staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -570,6 +573,30 @@ func GetFieldManagerFlag(cmd *cobra.Command) string {
 	return GetFlagString(cmd, "field-manager")
 }
 
+func GetValidationDirective(cmd *cobra.Command) (string, error) {
+	var validateFlag = GetFlagString(cmd, "validate")
+	b, err := strconv.ParseBool(validateFlag)
+	if err != nil {
+		switch validateFlag {
+		case cmd.Flag("validate").NoOptDefVal:
+			return metav1.FieldValidationStrict, nil
+		case "strict":
+			return metav1.FieldValidationStrict, nil
+		case "warn":
+			return metav1.FieldValidationWarn, nil
+		case "ignore":
+			return metav1.FieldValidationIgnore, nil
+		default:
+			return metav1.FieldValidationStrict, fmt.Errorf(`Invalid validate value (%v). Must be "true", "false", "strict", "warn", or "ignore".`, validateFlag)
```

Comment:
nit: take it or leave it:
```suggestion
			return metav1.FieldValidationStrict, fmt.Errorf(`invalid --validate option %q; must be one of: strict (or true), warn, ignore (or false)`, validateFlag)
```